### PR TITLE
Create physiotherapy-theory-and-practice.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/86539/physiotherapy-theory-and-practise#latest

### CODE REVIEW

File: physiotherapy-theory-and-practice.csl [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      <key macro="issued"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=". ">
+        <text macro="author"/>
+        <text macro="issued"/>
+        <choose>
+          <if type="article article-magazine article-newspaper article-journal review" match="any">
+            <text variable="title" suffix="."/>
```
Comment (bwiernik) [inline physiotherapy-theory-and-practice.csl:85]: Needs a space on the suffix here. Should probably be a group delimiter instead.

File: physiotherapy-theory-and-practice.csl [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            <group delimiter=" ">
+              <text variable="container-title" text-case="title" strip-periods="true"/>
+              <group delimiter=":">
+                <text variable="volume"/>
+                <text variable="page"/>
+              </group>
+            </group>
+          </if>
+          <else-if type="chapter paper-conference" match="any">
+            <text variable="title"/>
```
Comment (bwiernik) [inline physiotherapy-theory-and-practice.csl:95]: This needs a group delimiter between title and container information.

File: physiotherapy-theory-and-practice.csl [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            <group prefix="In: " delimiter=". ">
+              <text macro="editor"/>
+              <text variable="container-title" text-case="title" strip-periods="true"/>
+              <text macro="publisher"/>
+              <label variable="page" form="short" strip-periods="true" plural="never"/>
+              <text variable="page"/>
+            </group>
+          </else-if>
+          <else-if type="thesis">
+            <text variable="title" suffix="."/>
```
Comment (bwiernik) [inline physiotherapy-theory-and-practice.csl:105]: Suffixes need spaces. Should be group delimiters instead.

File: physiotherapy-theory-and-practice.csl [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            <text variable="genre" suffix="."/>
+            <text variable="publisher"/>
+          </else-if>
+          <else>
+            <text variable="title"/>
```
Comment (bwiernik) [inline physiotherapy-theory-and-practice.csl:110]: Same as above. Group delimiters.

**Review by bwiernik — CHANGES_REQUESTED** [review]

**Comment by POBrien333** [comment]
No idea what happened, but just reworked this. Loads of errors in it. Must've gotten distracted and then thought it was finished without thinking about it.

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**